### ISSUE TITLE
Call for maintainers

### ISSUE THREAD

**Issue by piranna** [chjj/blessed#254]
Similar to https://github.com/chjj/marked/issues/756, I would open a call for maintainers o collaborators on this project. I must to say in advance I'm sure if I would be a good candidate, both because I'm busy with NodeOS (and willing to have some collaborators there so I could have some life...) and because if I were one of the maintainers of blessed one of the first things I would do would be to move the widgets to independent projects and make a 0.5 release really minimalistic to allow more flexibility (like double buffering), and I know that chjj is not happy with both ideas in the current development status, but if the addition of collaborators would give some impetus to blessed I would definitely help to its development in some way or another.

**Comment by jamen** [comment]
I would be interested in helping maintain if that is a possibility (although I realize nothing is confirmed yet).  I love terminal-related projects, and I even composed interfaces like blessed with some raw methods before I knew it existed.  There is also lots of other JavaScript work on my profile if you are curious.  I could start out with some PRs if you have any ideas. Thanks. :>

**Comment by chazu** [comment]
I would love to help maintain as well, although I am by no means an expert in the codebase. I use blessed actively for personal projects and would relish the chance to improve this library.

**Comment by skerit** [comment]
Why a 0.5 release? The next minor version is 0.2 :)

Anyway, seems like a good idea. Blessed could really use some love.

**Comment by Kakise** [comment]
I am interested in maintening this project.

**Comment by piranna** [comment]
Thanks @Kakise :-) So where do you propose to start?

**Comment by Lambeaux** [comment]
This repo seems dead. Did anyone ever start a separate fork for support purposes?

**Comment by piranna** [comment]
> This repo seems dead. Did anyone ever start a separate fork for support purposes?

None that I would know, according to the [forks network](https://github.com/chjj/blessed/network) most of the alternatives are just pull-requests. Since I was the one that opened the issue I would start it, I'm a bit busy with NodeOS but as I've said before I have several ideas about how this module should move fordward. Do you agree?

**Comment by chazu** [comment]
@piranna What are your ideas? Although I lack the knowledge of the code base to be a maintainer at present 0 I've only used blessed casually in a couple of fun-time projects - I would like to see more activity around blessed in general, and would be willing to help make it happen. I think @chjj would generally sanction any reasonable plan to stimulate the ecosystem.

That being said, I would be willing to be a maintainer in the absence of any other volunteers - would also be willing to interview with other community members to assess my suitability.

I suggested in issue #256 the creation of an IRC channel - and chjj gave it his blessing on twitter - but I haven't created it yet as i'm not sure whether ##blessed or #blessed would be more appropriate - I think a single hash would be acceptable personally.

**Comment by piranna** [comment]
My main ideas are to split the widgets from the core on two or more different projects for clean-up both of code and APIs, and later move to a double buffer architecture where each widget draws on its own "screen" similar to how X11 and Plan9 `rio` works, that will make it easier to work with the widgets and also to do higher level tricks like transparency and alpha-blending, and finally add support CSS-like styles and HTML-like box model and CSS flexbox, so it will make it easier to program and use.

**Comment by chazu** [comment]
@piranna I've opened up ##blessed on freenode IRC for general discussion. Feel free to pop in and we can discuss more synchronously ideas for revitalizing this project.

One idea might be to set up an informal poll asking interested parties what they'd like to see happen with this project - better issue moderation, code cleanup, API restructuring, etc. Ideally a consensus could be reached about how to get things rolling without dividing potential contributors/users.

**Comment by piranna** [comment]
I agree on the pool, but I don't have availability for synchronous chat. I preffer to stick to async conversations over github issues.

**Comment by chazu** [comment]
@piranna Understandable, async is a good default.

In my mind the relevant questions re: revitalizing this project are who is willing to own a repo and coordinate activity (issues and PRs) while maintaining an open mind with regard to the direction of the project insofar as other contibutors are concerned?

I think it makes sense to defer to the existing contributors, so if @piranna or @Lambeaux want to create an organization or simply designate their fork as the de facto support fork, that ought to be sufficient, no?

**Comment by piranna** [comment]
I could be able to create a fork and an organization and manage the pull-requests, but due to lack of time I will not be able to lead the development. I could help to discuss the roadmap and the path of the project though, I think we need some guidelines about how it should move forward. What do you think?

**Comment by piranna** [comment]
[I have created a new organization and moved my fork there](https://github.com/blessed-2/blessed), whoever that wants to be a collaborator or an owner please tell me and I'll add him there.

**Comment by chazu** [comment]
@piranna Apologies for the long silence, mind adding me as collaborator?

**Comment by piranna** [comment]
> @piranna Apologies for the long silence, mind adding me as collaborator?

Done.

**Comment by OnkelTem** [comment]
Any updates?
